RealmVideo/RealmVideoViewController.swift

```diff
@@ -84,12 +84,19 @@ class RealmVideoViewController: UIViewController, UIWebViewDelegate {
         }
     }
     
+    deinit {
```

Comment:
Do we need to remove the observer starting from iOS 9? https://developer.apple.com/library/mac/releasenotes/Foundation/RN-Foundation/index.html#10_11NotificationCenter